.github/workflows/build_worker_images.yml

```diff
@@ -31,6 +31,13 @@ jobs:
           restore-keys: |
             pip-
       - run: pip install -r requirements.txt
+      - run: | 
+          sudo swapoff -a
+          sudo rm -f /swapfile
+          sudo rm -rf /opt/ghc
+          sudo apt clean
+          docker rmi $(docker image ls -aq)
+          df -h
       - run: python3 codalab_service.py build ${SERVICE} $([ -z "${CODALAB_DOCKER_USERNAME}" ] || echo "--push")
```

Comment:
Right now, it's pushing to a tag called HEAD, rather than the actual PR branch name. We should change this

![image](https://user-images.githubusercontent.com/1689183/120567510-fa561180-c3df-11eb-902a-6a665c631753.png)

Reply:
Update the github action file, should be working now: https://hub.docker.com/layers/codalab/default-cpu/update_images/images/sha256-ee7b6df243b8e51db27fccc8f1f125dee21bddff29fa21da1377b11b75c4acd4?context=explore